pkg/util/docker/global.go

```diff
@@ -17,12 +18,14 @@ import (
 )
 
 var (
-	globalDockerUtil     *DockerUtil
-	invalidationInterval = 5 * time.Minute
+	globalDockerUtil      *DockerUtil
+	globalDockerUtilMutex sync.RWMutex
```

Comment:
Should this just be a regular `sync.Mutex`?  It doesn't look like the read lock is used

Reply:
Definitively. Fixed!